lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -19,6 +20,7 @@ def initialize(input, suspended_action:)
         @check_mode = action_input[:check_mode]
         @tags = action_input[:tags]
         @tags_flag = action_input[:tags_flag]
+        @job_invocation_timeout = action_input[:job_invocation_timeout]
```

Comment:
why do we use different naming than `ScriptRunner`, which use `execution_timeout_interval`?
I'd love if we could keep the naming matching for the same purpose, or is there a reason for different name?

Reply:
@ezr-ondrej From what I understood the execution_timeout_interval is from the template invocation but if the value is nil in the template it does not mean that in the specific job invocation the timeout is nil as well. so I created another variable for the job_invocation timeout.

Reply:
well ... if I'm reading the theforeman/foreman_ansible#422 correctly, you're getting it from job_invocation anyway, I don't think `template_invocation` has timeout. All the user inputs are on job_invocation, so that would be weird.

Reply:
@ezr-ondrej you are right there is no need to add another parameter. I will fix this.